spec/controllers/users_controller_spec.rb

```diff
@@ -21,24 +21,73 @@
       it "redirects to the sign in page" do
         expect(response).to redirect_to sign_in_path
       end
-    end
 
-    context "email sending" do
+      it "makes the user follow the inviter" do
+        luke = Fabricate(:user)
+        invitation = Fabricate(:invitation, inviter: luke, recipient_email: "hans@falcon.com")
+        post :create, user: {
+          email: "hans@falcon.com",
+          full_name: "Hans Solo",
+          password: "password"
+        }, invitation_token: invitation.token
+
+        hans = User.where(email: "hans@falcon.com").first
+        expect(hans.follows?(luke)).to be_truthy
+      end
 
-      before { ActionMailer::Base.deliveries.clear }
+      it "make the inviter follow the user" do
+        luke = Fabricate(:user)
+        invitation = Fabricate(:invitation, inviter: luke, recipient_email: "hans@falcon.com")
+        post :create, user: {
+          email: "hans@falcon.com",
+          full_name: "Hans Solo",
+          password: "password"
+        }, invitation_token: invitation.token
+
+        hans = User.where(email: "hans@falcon.com").first
+        expect(luke.follows?(hans)).to be_truthy
+      end
 
+      it "expires the invitation upon acceptance" do
+        luke = Fabricate(:user)
+        invitation = Fabricate(
+          :invitation,
+          inviter: luke,
+          recipient_email: "hans@falcon.com"
+        )
+        post :create, user: {
+          email: "hans@falcon.com",
+          full_name: "Hans Solo",
+          password: "password"
+        }, invitation_token: invitation.token
+        expect(Invitation.first.token).to be_nil
+      end
+    end
+
+    context "email sending" do
       it "sends out the email given valid inputs" do
-        post :create, user: Fabricate.attributes_for(:user, email: "example@example.com", full_name: "The Wizard of Id")
+        post :create,
+          user: Fabricate.attributes_for(:user,
+                                         email: "example@example.com",
+            full_name: "The Wizard of Id")
         expect(ActionMailer::Base.deliveries.last.to).to eq(["example@example.com"])
       end
 
       it "email sent out has users name" do
-        post :create, user: Fabricate.attributes_for(:user, email: "example@example.com", full_name: "The Wizard of Id")
+        post :create,
+          user: Fabricate.attributes_for(
+            :user, email: "example@example.com", full_name: "The Wizard of Id"
+        )
         expect(ActionMailer::Base.deliveries.last.body).to include("The Wizard of Id")
       end
 
       it "email does not send given invalid inputs" do
-        post :create, user: { email: "example@example.com", full_name: "The Wizard of Id", password: "" }
+        post :create,
+          user: {
+          email: "example@example.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.